tests/publiccloud/prepare_tools.pm

```diff
@@ -122,6 +122,25 @@ EOT
     validate_script_output("terraform -version", qr/$terraform_version/);
     record_info('Terraform', script_output('terraform -version'));
 
+    # Ansible install with pip
+    # Default version is chosen as low as possible so it run also on SLE12's
+    # ANSIBLE_CORE_VERSION should be set only if the different then default one need to be used
+    my $ansible_version = get_var('ANSIBLE_VERSION', '4.10.0');
+    my $ansible_core_version = get_var('ANSIBLE_CORE_VERSION');
+    my $ansible_install_log = '/tmp/ansible_install.log';
+
+    assert_script_run("curl https://bootstrap.pypa.io/get-pip.py -o /tmp/get-pip.py", timeout => 240);
```

Comment:
I don't think we need to get pip from bootstrap.pypa.io, if it's already installed in line 74, no?
(same for the next line)

Reply:
Because in hurry I deleted the other part of the statement! Ty for the catch